lib/app_server.py

```diff
@@ -3,10 +3,14 @@
 import os
 import shutil
 import sys
+import signal
 
 from gevent.subprocess import Popen, PIPE
+from gevent import Timeout
```

Comment:
It is unused. Please, check `make lint` before sending a PR.

Reply:
Sorry, forgot to update the PR from the staging branch.